homeassistant/components/heatzy/switch.py

```diff
@@ -0,0 +1,79 @@
+"""Sensors for Heatzy."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from heatzypy.exception import HeatzyException
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import HeatzyDataUpdateCoordinator
+from .const import CONF_ALIAS, CONF_ATTR, CONF_ATTRS, CONF_LOCK, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set the sensor platform."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    entities = [
+        LockSwitchEntity(coordinator, unique_id)
+        for unique_id in coordinator.data.keys()
+    ]
+    async_add_entities(entities)
+
+
+class LockSwitchEntity(CoordinatorEntity[HeatzyDataUpdateCoordinator], SwitchEntity):
+    """Lock Switch."""
+
+    entity_category = EntityCategory.CONFIG
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, coordinator: HeatzyDataUpdateCoordinator, unique_id: str
+    ) -> None:
+        """Initialize switch."""
+        super().__init__(coordinator)
+        self._attr_unique_id = unique_id
+        self._attr_name = "Lock switch {}".format(
+            coordinator.data[unique_id][CONF_ALIAS]
+        )
```

Comment:
Could use an f-string for brevity.

```suggestion
        self._attr_name = f"Lock switch {coordinator.data[unique_id][CONF_ALIAS]}"
```